frontend/src/components/TemplateFlow.js

```diff
@@ -42,9 +36,43 @@ function renderResult(checkResult) {
     })
   })
 
-  return errors
+  if (checkResult.error) {
+    return <>
+      <pre>
+        {checkResult.error.message}
+      </pre>
+      {errors}
+    </>
+  }
+  else {
+    return <>
+      {errors}
+    </>
+  }
+}
+
+function renderResultOK(checkResult) {
+  const results = []
+  const columns = []
+
+  checkResult.diff_headers?.forEach((diff_header, index) => {
+    columns.push({title: diff_header, dataIndex: `${index}`, key: `${index}`})
+  })
+
+  checkResult.rows.forEach((row, index) => {
+    const row_data = {}
+    row.diff.forEach((diff, diff_index) => {
+      row_data[`${diff_index}`] = stringToHTML(diff)
```

Comment:
`row_data[diff_index]`? or `row_data[String(diff_index)]`?

Reply:
String, I just changed it to `column-${diff_index}` to clarify what it is